pkg/registry/apis/provisioning/controller/finalizers.go

```diff
@@ -124,6 +121,34 @@ func (f *finalizer) processExistingItems(
 	return nil
 }
 
+func getPatchedAnnotations(item *provisioning.ResourceListItem) []byte {
```

Comment:
Not super happy with the string manipulation stuff here - another idea to avoid other deps would be to use something like
```go
type jsonPatchOperation struct {
	Op    string `json:"op"`
	Path  string `json:"path"`
}
```

and use json marshalling - wdyt?

Reply:
I think that's a good idea :)

Reply:
what happened with your attempt?

Reply:
What do you mean?